### ISSUE TITLE
h.default.findWhere is not a function

### ISSUE THREAD

**Issue by Larry850806** [dockersamples/docker-swarm-visualizer#98]
I use latest docker image, which is created about an hour ago. It's image id is `ab6b6bda4129`. It shows the error on the webpage

![screen shot 2017-12-01 at 8 58 41 am](https://user-images.githubusercontent.com/10403741/33463036-130ba3a8-d676-11e7-853c-ae67f9d215ca.png)

**Comment by ManoMarks** [comment]
@Larry850806 can you try pulling again? I'm unable to reproduce the issue. If that doesn't fix it, please post the steps you're taking so we can reproduce

**Comment by Larry850806** [comment]
I use visualizer as a service. This is part of my `docker-compose.yml`

```yml
  visualizer:
    image: dockersamples/visualizer:stable
    volumes: 
      - /var/run/docker.sock:/var/run/docker.sock
    ports: 
      - 8080:8080
    deploy:
      placement:
        constraints:
          - node.role == manager 
```

Everything is ok when I use `dockersamples/visualizer:stable`. But when I use `dockersamples/visualizer:latest`. Webpage doesn't show services but only error message in devtool.

**Comment by ManoMarks** [comment]
Fixed in #100

**Comment by chabad360** [comment]
this issue is back
